modules/elements-color-picker/assets/js/editor/commands/enter-preview.js

```diff
@@ -2,8 +2,22 @@ import CommandBase from 'elementor-api/modules/command-base';
 
 export class EnterPreview extends CommandBase {
 	apply( args ) {
+		const { container, kit } = this.component.currentPicker;
+
 		// Silent
-		this.component.currentPicker.container.settings.set( this.component.currentPicker.control, args.value );
-		this.component.currentPicker.container.view.renderUI();
+		container.settings.set( control, args.value );
+
+		const { view } = container;
+
+		if ( view?.renderUI ) {
+			view.renderUI();
+		}
+
+		if ( kit ) {
```

Comment:
An empty object is `true`, see assets/dev/js/editor/controls/color.js:90

Reply:
Oops, meant to write null..